hello-html/index.ts

```diff
@@ -0,0 +1,41 @@
+export async function execute(context: Context): Promise<Output> {
+    const stream = await context.openWriteText(context.parameters.response, "utf-8", responseOptions);
+    const writer = stream.getWriter();
+    await writer.write(html);
+    await writer.close();
+}
+
+const responseOptions = {
+    headers: {
+        "Content-Type": "text/html; charset=utf-8",
+    },
+};
+
+const html = `<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Hello World!</title>
+	</head>
+	<body>
+		<h1>Hello World!</h1>
+	</body>
+</html>
+`;
+
+export function getDescription() {
```

Comment:
To getDescription bych dal nahoru, ať je to konzistentní ve scriptech.

Reply:
Opraveno.